mdx/guides/integrate-instant.mdx

```diff
@@ -0,0 +1,208 @@
+| `onSuccess`                      | a function handler that is called when the token purchase through 0x Instant is complete. The function handler requests one argument: the transaction hash.                                                                                                                                                                                                                                                                                           |
+| `onClose`                        | a function handler that is called when the Instant overlay is closed. The function handler does not request any argument                                                                                                                                                                                                                                                                                                                              |
+
+# Examples
+
+
+## With token list
+
+Check all others token lists on [List Repository](https://tokenlists.org/) or create one following the [specification](https://github.com/Uniswap/token-lists) and host it on Github
+
+```typescript
+zeroExInstant.render(
+    {
+        tokenList: 'https://tokens.coingecko.com/uniswap/all.json',
+    },
+    'body',
+);
+```
+
+## Providing your own provider
+
+This will give you more control over what provider is passed in and where RPC calls are directed
+
+```typescript
+zeroExInstant.render(
+    {
+        provider: window.ethereum,
+        walletDisplayName: 'Trust Wallet',
+    },
+    'body',
+);
+```
+
+## Providing a Default Token In
+
+```typescript
+zeroExInstant.render(
+    {
+       defaultSelectedTokenIn: '0xe41d2489571d322189246dafa5ebde1f4699f498'
+    },
+    'body',
+);
+```
+
+## Example of Swap Token In by Token out with defined amount
+
+```typescript
+zeroExInstant.render(
+    {
+       defaultSelectedTokenIn: '0xe41d2489571d322189246dafa5ebde1f4699f498',
+       defaultSelectedTokenOut: '0x1985365e9f78359a9b6ad760e32412f4a445e862',
+       defaultAmountTokenIn: 10000000000,
+    },
+    'body',
+);
+```
+
+## Providing your own callback handler
+
+```typescript
+zeroExInstant.render(
+    {
+        onSuccess: txHash => {
+            console.log('Success! Transaction hash is: ' + txHash);
+        },
+    },
+    'body',
+);
+```
+
+# Learn about affiliate fees
+
+As an end host of 0x Instant, you can charge users a fee on all trades made through Instant with the `affiliateFee` option. Simply specify an ethereum address and feePercentage (up to 5%), and a percentage of each transaction will be deposited into the specified address (denominated in bought tokens).
+
+Example: 3% of transaction volume (in ETH) will be deposited into 0x50ff5828a216170cf224389f1c5b0301a5d0a230
+
+```typescript
+zeroExInstant.render(
+    {
+        affiliateInfo: {
+            feeRecipient: '0x50ff5828a216170cf224389f1c5b0301a5d0a230',
```

Comment:
I hope this is not a default in the code!

Reply:
The default is the ZERO Address